src/QMCHamiltonians/NonLocalECPComponent.cpp

```diff
@@ -550,17 +550,17 @@ NonLocalECPComponent::RealType NonLocalECPComponent::evaluateOneWithForces(Parti
     {
       //Note.  Because we are computing "forces", there's a -1 difference between this and
       //direct finite difference calculations.
-      lsum           += vrad[l] * lpol[angpp_m[l]] * psiratio[j];
-      gradpotterm_   += vgrad[l] * lpol[angpp_m[l]] * psiratio[j];
+      lsum += vrad[l] * lpol[angpp_m[l]] * psiratio[j];
+      gradpotterm_ += vgrad[l] * lpol[angpp_m[l]] * psiratio[j];
       gradlpolyterm_ += vrad[l] * dlpol[angpp_m[l]] * cosgrad[j] * psiratio[j];
-      gradwfnterm_   += vrad[l] * lpol[angpp_m[l]] * wfngrad[j];
+      gradwfnterm_ += vrad[l] * lpol[angpp_m[l]] * wfngrad[j];
```

Comment:
These changes will just be reverted again by auto-format, but I guess it is no big deal.

Reply:
At the moment I don’t think so I didn’t have the bandwidth to fight for my consistent assignment operator PR to clang-format. I think we just have to let these go for now.

Reply:
Actually this is changed by clang-format not me.